src/main/java/com/agilethought/internship/university/controller/UniversityController.java

```diff
@@ -42,4 +40,15 @@ public UniversityController(UniversityService service){
 		log.info("UniversityController.postCourses - operation response {}", response);
 		return new ResponseEntity<>(response,HttpStatus.CREATED);
 	}
+
+	@PutMapping(value = "/course/{id}")
+	@ResponseStatus(HttpStatus.OK)
+	public ResponseEntity<UpdateCourseResponse> putOperation(
+			@RequestBody UpdateCourseRequest request, @PathVariable String id) {
+		log.info("Calling Put operation request {}", request);
+		UpdateCourseResponse response = service.Updatecourse(request, id);
+		log.info("Calling Put operation response {}", response);
```

Comment:
Same here, it is not a good practice log response information, add just a singel comment that PUT operation was successful